selectors/rhs.jsx

```diff
@@ -41,3 +41,15 @@ export const getSelectedPost = createSelector(
         };
     }
 );
+
```

Comment:
Looks like line 16 (`return state.views.rhs.selectedPostChannelId;`) was changed to `channelId`

Reply:
Good catch 👍